lms/djangoapps/shoppingcart/tests/test_pdf.py

```diff
@@ -0,0 +1,213 @@
+"""
+Tests for Pdf file
+"""
+from datetime import datetime
+from django.test.utils import override_settings
+from django.conf import settings
+import unittest
+from io import BytesIO
+from shoppingcart.pdf import PDFInvoice
+from shoppingcart.utils import parse_pages
+
+
+class TestPdfFile(unittest.TestCase):
+    """
+    Unit test cases for pdf file generation
+    """
+    def setUp(self):
+        self.context = {
+            'items_data': [self.get_item_data(1)],
+            'id': str(1),
+            'date': datetime.now(),
+            'is_invoice': False,
+            'total_cost': 1000,
+            'payment_received': 1000,
+            'balance': 0,
+        }
+        self.pdf_buffer = BytesIO()
+
+    def get_item_data(self, index, discount=0):
+        """
+        return the dictionary with the dummy data
+        """
+        return {
+            'item_description': 'Course %s Description' % index,
+            'quantity': index,
+            'list_price': 10,
+            'discount': discount,
+            'item_total': 10
+        }
+
+    @override_settings(
+        PDF_RECEIPT_DISCLAIMER_TEXT="THE SITE AND ANY INFORMATION, CONTENT OR SERVICES MADE AVAILABLE ON OR THROUGH "
+                                    "THE SITE ARE PROVIDED \"AS IS\" AND \"AS AVAILABLE\" WITHOUT WARRANTY OF ANY KIND"
+                                    " (EXPRESS, IMPLIED OR OTHERWISE), INCLUDING, WITHOUT LIMITATION, ANY IMPLIED"
+                                    " WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND"
+                                    " NON-INFRINGEMENT, EXCEPT INSOFAR AS ANY SUCH IMPLIED WARRANTIES MAY NOT BE"
+                                    " DISCLAIMED UNDER APPLICABLE LAW.",
```

Comment:
I'd prefer if these were defined as constants in the test case.  Please use parentheses to avoid over-indenting:
```
(
    "Lorem "
    " ipsum"
)
```